src/store/auth.js

```diff
@@ -39,8 +41,13 @@ export default {
     socket_updateUserData(/* state */) {
       console.log('socket_updateUserData');
     },
-    socket_login(/* state */) {
-      console.log('socket_login');
+    socket_login(state, token) {
```

Comment:
Use the "outside a vue file" syntax from https://v1.quasar-framework.org/quasar-plugins/notify#Notify-API here to notify with the response's `value` if `!isSuccessful.